### PR TITLE
Improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
# Stats

Baseline      1101 API locations with examples

Provider:     google-beta
Success rate: 94.94% (14457/15228)

Converted 93.74% of csharp examples (2379/2538)
Converted 93.70% of go examples (2378/2538)
Converted 97.64% of java examples (2478/2538)
Converted 93.74% of python examples (2379/2538)
Converted 93.74% of typescript examples (2379/2538)
Converted 97.08% of yaml examples (2464/2538)

Experimental  1101 API locations with examples

Provider:     google-beta
Success rate: 96.77% (14880/15377)

Converted 96.73% of csharp examples (2481/2565)
Converted 96.69% of go examples (2480/2565)
Converted 96.69% of java examples (2480/2565)
Converted 96.73% of python examples (2481/2565)
Converted 96.73% of typescript examples (2481/2565)
Converted 97.06% of yaml examples (2477/2552)

### CODE REVIEW

File: .ci-mgmt.yaml [hunk 1]
```diff
@@ -20,12 +20,18 @@ plugins:
...
   - name: tls
     version: "4.11.1"
   - name: http
     version: "0.0.1"
   - name: time
     version: "0.0.15"
+  - name: terraform
+    version: "1.0.15"
+    kind: converter
+team: ecosystem
```
Comment (iwahbe) [inline .ci-mgmt.yaml:34]:
```suggestion
```

I don't think we use this var anymore. Has that changed?

File: sdk/dotnet/AccessApproval/GetOrganizationServiceAccount.cs [hunk 2]
```diff
@@ -39,7 +39,7 @@ public static class GetOrganizationServiceAccount
         /// 
         ///     var iam = new Gcp.Kms.CryptoKeyIAMMember("iam", new()
         ///     {
-        ///         CryptoKeyId = google_kms_crypto_key.Crypto_key.Id,
+        ///         CryptoKeyId = cryptoKey.Id,
```
Comment (iwahbe) [inline sdk/dotnet/AccessApproval/GetOrganizationServiceAccount.cs:42]: Nice.

File: sdk/go/gcp/workstations/workstationCluster.go [hunk 3]
```diff
@@ -171,7 +177,7 @@ import (
 //				Annotations: pulumi.StringMap{
 //					"label-one": pulumi.String("value-one"),
 //				},
-//			}, pulumi.Provider(google_beta))
```
Comment (iwahbe) [inline sdk/go/gcp/workstations/workstationCluster.go:174]: Why did we used to specify an explicit provider, but we don't anymore?

**Review by iwahbe — APPROVED** [review]

File: sdk/go/gcp/workstations/workstationCluster.go (hunk 3 above)
Comment (t0yv0) [reply sdk/go/gcp/workstations/workstationCluster.go:174]: I'm not sure but this looks like a fix. The variable used to be unbound. I think the change is for good.

**Pull request merged**